application/controllers/EnrollmentForm.php

```diff
@@ -250,16 +245,12 @@ public function stripePaymentSuccessful()
             // Highlight this row since it is paid
             $this->GoogleSheets_Model->highlightRow($row ,[0.69803923, 0.8980392, 0.69803923]);
 
-            // Send a confirmation email only if it wasn't previously highlighted
-            if (!$alreadyHighlighted) {
-                $this->load->model('Email_Model');
-                $this->Email_Model->sendConfirmationEmail($data['email'], "online", $this->eventData);
-            }
+            $this->load->model('Email_Model');
+            $this->Email_Model->sendConfirmationEmail($data['email'], "online", $this->eventData);
```

Comment:
We should send a confirmation email regardless of past highlighting

Reply:
The reason to this one, if I remembered correctly: If you refresh the page on payment successful, it resends the email for you, which it shouldn't. This check prevents that.

Reply:
Ah awesome, I'll put those changes back and add a comment. I saw the comments around the highlight section, but it didn't make sense to me – now it does!